lib/saml/user.rb

```diff
@@ -43,6 +43,18 @@ def self.context_key(authn_context)
       UNKNOWN_CONTEXT
     end
 
+    # we use this for statsd tags
+    def account_type
+      case authn_context
+      when 'myhealthevet'
+        user_attributes.mhv_account_type
+      when 'dslogon'
+        user_attributes.dslogon_assurance
+      else
+        user_attributes.loa[:current]
```

Comment:
maybe we should use the actual raw loa we get from id.me, not the derived loa[:current], i realize this could be null though, but we'll know what the business logic is around it.